src/refactor_nrepl/middleware.clj

```diff
@@ -35,6 +35,11 @@
   `(with-errors-being-passed-on ~transport ~msg
      (transport/send ~transport (response-for ~msg ~(apply hash-map kv)))))
 
+(defn format-response [{:keys [format] :as msg} response]
```

Comment:
I think a better name here would be `serialize-response` or something along those lines.  To me `format-response` makes me think about pretty printing more than producing different data formats.